CriticalMass/LocationManager.swift

```diff
@@ -19,6 +19,9 @@ class LocationManager: NSObject, CLLocationManagerDelegate, LocationProvider {
             case .restricted,
                  .denied:
                 return .denied
+            @unknown default:
```

Comment:
I think this is fine but I don't know if any new cases will be added to authorizationStatus. Or what is the reason you added it here?

Reply:
Its not Markes as a frozen enumeration (not sure if they forgot it or wanna add cases in the future) so not handling the case gives an Warnung with swift 5 https://docs.swift.org/swift-book/ReferenceManual/Statements.html

Reply:
Interesting 👍